src/pages/LandingPage/LandingPage.jsx

```diff
@@ -26,7 +27,10 @@ const LandingPage = () => {
 
   const handleClick = () => {
     console.log('Shape shifter button clicked! Current shape: ', shape); //<-- console.log("someText", someVariable) super nice way to check if buttons and functions are working
-    const chooseRandomShape = Math.floor(Math.random() * possibleShapes.length);
+    let chooseRandomShape = prevShape
+    while(chooseRandomShape === prevShape){
+        chooseRandomShape = Math.floor(Math.random() * possibleShapes.length);
+    } prevShape = chooseRandomShape
```

Comment:
This is a very good use of a while loop! The solution is correct for all cases except the first click, as the previousShape is a string it will never match anything in the possibleShapes array. So if the random shape is "torus" same as the initial starting shape, nothing will happen. 

I have added a commit with a fix and some refactoring

Reply:
Thank you.